src/math.jl

```diff
@@ -1,4 +1,5 @@
 for (type, base_type, _) in ABSTRACT_QUANTITY_TYPES
+    div_base_type = type == AbstractGenericQuantity ? Number : base_type
```

Comment:
```suggestion
    div_base_type = type === AbstractGenericQuantity ? Number : base_type
```
Tiny nit: I think convention is to use `===` for comparing types. (Maybe because it's a little safer since `==` is technically overloadable even on types?)

Reply:
```suggestion
    div_base_type = type <: Number ? base_type : Number
```
Moreover, could this check be more elegantly written like this, or something similar?